app/src/main/java/com/example/appmarvel/di/ViewModelsModule.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.appmarvel.di
+
+import com.example.appmarvel.mvvm.model.CharacterListModel
+import com.example.appmarvel.mvvm.viewmodel.CharactersListViewModel
+import com.example.appmarvel.mvvm.viewmodel.MainViewModel
+import com.example.appmarvel.mvvm.viewmodel.SplashScreenViewModel
+import org.koin.androidx.viewmodel.dsl.viewModel
+import org.koin.dsl.module
+
+object ViewModelsModule {
+    val viewModelsModule = module {
+
+        viewModel { SplashScreenViewModel() }
+        viewModel { MainViewModel() }
+        viewModel { CharactersListViewModel(get()) }
+        single { CharacterListModel() }
```

Comment:
1. This should be in a separated module
2. Why it is a single? I think that a factory should be better

Reply:
Done